types/node/net.d.ts

```diff
@@ -532,6 +532,12 @@ declare module "net" {
          * @since v16.5.0
          */
         keepAliveInitialDelay?: number | undefined;
+        /**
+         * Optionally overrides all net.Sockets' readableHighWaterMark and writableHighWaterMark.
+         * Default: @see stream.getDefaultHighWaterMark().
+         * @since v18.17.0
```

Comment:
```suggestion
         * @default See [stream.getDefaultHighWaterMark()](https://nodejs.org/docs/latest-v22.x/api/stream.html#streamgetdefaulthighwatermarkobjectmode).
         * @since v18.17.0, v20.1.0
```